src/main/java/org/zaproxy/zap/extension/jwt/fuzzer/ExtensionJWTFuzzer.java

```diff
@@ -0,0 +1,139 @@
+package org.zaproxy.zap.extension.jwt.fuzzer;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.control.Control;
+import org.parosproxy.paros.extension.Extension;
+import org.parosproxy.paros.extension.ExtensionAdaptor;
+import org.parosproxy.paros.extension.ExtensionHook;
+import org.parosproxy.paros.extension.ViewDelegate;
+import org.parosproxy.paros.network.HttpMessage;
+import org.zaproxy.zap.extension.fuzz.ExtensionFuzz;
+import org.zaproxy.zap.extension.fuzz.httpfuzzer.processors.FuzzerHttpMessageScriptProcessorAdapterUIHandler;
+import org.zaproxy.zap.extension.fuzz.messagelocations.MessageLocationReplacers;
+import org.zaproxy.zap.extension.httppanel.component.all.request.RequestAllComponent;
+import org.zaproxy.zap.extension.httppanel.component.split.request.RequestSplitComponent;
+import org.zaproxy.zap.extension.httppanel.component.split.request.RequestSplitComponent.ViewComponent;
+import org.zaproxy.zap.extension.jwt.JWTI18n;
+import org.zaproxy.zap.extension.jwt.fuzzer.messagelocations.JWTMessageLocationReplacerFactory;
+import org.zaproxy.zap.extension.jwt.fuzzer.ui.JWTFuzzPanelViewFactory;
+import org.zaproxy.zap.extension.script.ExtensionScript;
+import org.zaproxy.zap.view.HttpPanelManager;
+
+public class ExtensionJWTFuzzer extends ExtensionAdaptor {
+
+    private static final List<Class<? extends Extension>> DEPENDENCIES;
+
+    private JWTFuzzerHandler jwtFuzzerHandler;
+    private JWTMessageLocationReplacerFactory jwtMessageLocationReplacerFactory;
+
+    static {
+        List<Class<? extends Extension>> dependencies = new ArrayList<>(1);
+        dependencies.add(ExtensionFuzz.class);
+        DEPENDENCIES = Collections.unmodifiableList(dependencies);
+    }
+
+    public ExtensionJWTFuzzer() {
+        super();
+    }
+
+    @Override
+    public String getDescription() {
+        return Constant.messages.getString("fuzz.httpfuzzer.description");
+    }
+
+    @Override
+    public List<Class<? extends Extension>> getDependencies() {
+        return DEPENDENCIES;
+    }
+
+    @Override
+    public String getAuthor() {
+        return Constant.ZAP_TEAM;
+    }
+
+    @Override
+    public boolean supportsDb(String type) {
+        return true;
+    }
+
+    @Override
+    public void init() {
+        jwtFuzzerHandler = new JWTFuzzerHandler();
+        jwtMessageLocationReplacerFactory = new JWTMessageLocationReplacerFactory();
+        MessageLocationReplacers.getInstance()
+                .addReplacer(HttpMessage.class, jwtMessageLocationReplacerFactory);
+    }
+
+    @Override
+    public void initView(ViewDelegate view) {
+        super.initView(view);
+
+        ExtensionScript extensionScript =
+                Control.getSingleton().getExtensionLoader().getExtension(ExtensionScript.class);
+        HttpPanelManager panelManager = HttpPanelManager.getInstance();
+        panelManager.addRequestViewFactory(
```

Comment:
This also adds JWT view to Http Fuzzer. Not sure if there is any way to handle this. Need to look back. 
If we don't handle this then in Http Fuzzer it will look similar to JWT Fuzzer but working differs a lot wherein JWT fuzzer does many other things like Signing the token and chaning a field inside the token but http fuzzer just replaces the entire token @kingthorin @thc202 please suggets.

Reply:
This is a good point, will check.

Reply:
This might not be an issue as same menu item is now used for HttpFuzzer and JWT fuzzer